### PR TITLE
211/snapshot to disk

### LINKED ISSUE #173: Implement balance snapshotting to Disk
Implement balance snapshotting to permanent storage on disk

check out https://github.com/Polkadex-Substrate/polkadexTEE-worker/blob/develop/worker/src/polkadex_db.rs for an example (or maybe extend current version?)

### PR DESCRIPTION
**Pull request by haerdib**
Features interval disk snapshotting of the following storages:
- balances mirror
- nonce mirror
- oderbook mirror

Partially solves #211 
closes #173

### CODE REVIEW

File: worker/src/constants.rs [hunk 1]
```diff
@@ -40,3 +40,9 @@ pub static EXTRINSIC_MAX_SIZE: usize = 4196;
 pub static STATE_VALUE_MAX_SIZE: usize = 1024;
 // Iterator for RocksDB returns these many elements in a single yield
```
Comment (arrudagates) [inline worker/src/constants.rs]: I guess it's just a comment, but as we are not using RocksDB anymore this could be changed.

**Review by arrudagates — APPROVED** [review]
LGTM

File: worker/src/constants.rs (hunk 1 above)
Comment (haerdib) [reply worker/src/constants.rs]: Fixed

**Review by simonsso — APPROVED** [review]

**Comment by simonsso** [comment]
Please squash merge this as soon as it have been manually verified

**Comment by haerdib** [comment]
> Please squash merge this as soon as it have been manually verified

Interval loop is working (manually verified the loop entering the write to disk function). I was not able to test it yet with data within the mirrors (due to issue #236), but the writing to disk storage was tested with unit tests.

**Pull request merged**